python/download_dashboard_pdf.py

```diff
@@ -46,15 +52,22 @@ def get_dashboard(title: str) -> Optional[models.Dashboard]:
 
 
 def download_dashboard(
-    dashboard: models.Dashboard, style: str, width: int, height: int
+    dashboard: models.Dashboard,
+    filters: Optional[Dict[str, str]],
+    style: str,
+    width: int,
+    height: int,
```

Comment:
I know it's an artifact of how we parse arguments and call this function but maybe let's move `filters` to the end since it's `Optional` and default it to `None`. As a standalone this function reads kind of funny if you want to call it separately:

```python
download_dashboard(dashboard, None, "tiled", 545, 842)
```

In fact, even though it may be redundant, maybe we should encode those default argument values in this function anyway.